node/tests/swamp/swamp.go

```diff
@@ -0,0 +1,166 @@
+package swamp
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"net"
+	"testing"
+
+	"github.com/ipfs/go-log/v2"
+	"github.com/libp2p/go-libp2p-core/host"
+	mocknet "github.com/libp2p/go-libp2p/p2p/net/mock"
+	ma "github.com/multiformats/go-multiaddr"
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/celestia-node/core"
+	"github.com/celestiaorg/celestia-node/libs/keystore"
+	"github.com/celestiaorg/celestia-node/node"
+	"github.com/celestiaorg/celestia-node/node/p2p"
+)
+
+var blackholeIP6 = net.ParseIP("100::")
+
+// TODO(@Bidon15):
+// in case ConnectPeers from Swamp.Network is utilized to connect nodes between each other
+// then p2p_exchange.go line 55 close() will panic due to being closed from dial_sync.go(go-libp2p-swarm lib)
```

Comment:
Can you link to an exact commit and line on GitHub here? This documentation is brittle.

Reply:
I've made a separate issue #350 as the links look a lil bit ugly in the code comments section